cli/crates/backend/src/api/consts.rs

```diff
@@ -7,6 +7,7 @@ pub const TAR_CONTENT_TYPE: &str = "application/x-tar";
 pub const GRAFBASE_ACCESS_TOKEN_ENV_VAR: &str = "GRAFBASE_ACCESS_TOKEN";
 
 const API_URL: &str = "https://api.grafbase.com/graphql";
+const DASHBOARD_URL: &str = "https://app.grafbase.com";
```

Comment:
@fbjork IIRC we left the auth URL as is on purpose but we could use this for that as well

Reply:
@fbjork any opinion here?